docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -44,10 +44,10 @@ Change directories to *FSNetCore* and start adding projects to the solution fold
 Use the `dotnet new` command, create a Class Library project in the **src** folder named Library. 
 
 ```bash
-dotnet new classlib -lang F# -o src/Library 
+dotnet new lib -lang F# -o src/Library 
```

Comment:
@mairaw issue is, in F# nobody create `classlib` (class are A LOT less used in F#) but Libraries (so `lib`).
Seems silly, but is annoying to read and longer :D
Also is aligned to Visual Studio templates where is add new `F# library` not class lib

![image](https://user-images.githubusercontent.com/147243/27100124-4df89da6-507d-11e7-9c48-d6ce9a06d949.png)